test/recipes/80-test_ca.t

```diff
@@ -71,6 +72,138 @@ SKIP: {
        "Signing SM2 certificate request");
 }
 
+test_revoke('notimes',                    {});
```

Comment:
I probably would have just added a simple extra parameter here that is the expected value 1 or 0 instead of doing the regex..

Reply:
Good point, I probably overthought that a bit. I've replaced that with a single `should_succeed` key.